lib/istio/addon/gateway/detail/edit/route.js

```diff
@@ -0,0 +1,10 @@
+import Route from '@ember/routing/route';
+import { hash } from 'rsvp';
+
+export default Route.extend({
+  model() {
+    const original = this.modelFor('gateway.detail').gateway;
+
+    return hash({ gateway: original.clone(),  });
```

Comment:
I dont think you need hash here as you're not doing any promises. Its fine to return the object without the hash.